src/DotNetDockerTest.csproj

```diff
@@ -1,29 +1,32 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <OutputPath>lib\</OutputPath>
-    <NuspecFile>DotNetDockerTest.nuspec</NuspecFile>
-    <NuspecProperties>Version=$(Version)</NuspecProperties>
+	  <TargetFrameworks>netcoreapp2.1;net5.0;net6.0</TargetFrameworks>
```

Comment:
Should we still support .NET Core 2.1? It's been out of support since August 21, 2021. Maybe we want `netcoreapp3.1` instead?

Reply:
Question: why do we build different assemblies for .NET Core, .NET 5, and .NET 6? Are we taking advantage of some distinct features from them, or can we instead target just one framework?

Reply:
We should probably remove 2.1 by now. I would use netcoreapp3.1 but I think that there was a change to the docker image which broke this and I don't know if they backported the fix (I'll check and if that is the case bump the version).

Reply:
I've confirmed that the current microsoft dotnet 3.1 docker images doesn't have the property that allows me to just call dotnet vstestconsole. Happy to remove 2.1 now given that it is out of support but I think I'll probably stick with net5 and net6 unless I can get the change backported.